AdvancedDLSupport/NativeLibraryBuilder.cs

```diff
@@ -497,16 +522,18 @@ private Type GenerateInterfaceImplementationType
                 c => c.HasCustomAttribute<AnonymousConstructorAttribute>()
             );
 
+            var anonConstructorParams = anonymousConstructor.GetParameters();
```

Comment:
Avoid unneccesary contractions. Rename to constructorParameters.

Reply:
Thing is, its not a contradiction...

Reply:
A contraction, not a contradiction. Words should not be shortened in member and variable names.